final_project/final_project/settings.py

```diff
@@ -115,7 +115,7 @@
 
 USE_I18N = True
 
-USE_TZ = True
```

Comment:
W waszym projekcie nie jest to może teraz turbo ważne, ale normalnie chcemy używać timezone zawsze bo to się wszystko pomiesza, szczególnie jak macie apkę, gdzie jest coś a'la schedule